packages/flutter/test/material/chip_test.dart

```diff
@@ -307,7 +307,7 @@ void main() {
     expect(getIconData(tester).color?.value, 0xffffffff);
     expect(getIconData(tester).opacity, null);
     expect(getIconData(tester).size, null);
-    expect(getLabelStyle(tester).style.color?.value, 0xffffffff);
+    expect(getLabelStyle(tester, 'Chip A').style.color?.value, 0xdeffffff);
```

Comment:
@HansMuller Can you double-check that this is actually what the color should be in this case? It matches what the ChoiceChip already asserts.

Reply:
Yes, it's correct.